.github/ISSUE_TEMPLATE/support_request.md

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
This can potentially bring a lot of work...

Reply:
Agreed. If we find that it's too much for us to support then we can communicate that and remove this template.